src/04_features/filter/filter.scss

```diff
@@ -0,0 +1,51 @@
+
+.filter{
+    &__subject{
+        margin-bottom: 14px;
+        width: 194px;
+        height: 44px;
```

Comment:
do not use fconstant height and width. stretch layout
for exp:
wodth: 100%;
max-width: 200px;